cmd/tailscale/cli/network-lock.go

```diff
@@ -702,14 +704,17 @@ func runNetworkLockLog(ctx context.Context, args []string) error {
 	if err != nil {
 		return fixTailscaledConnectError(err)
 	}
-	if nlLogArgs.json {
-		enc := json.NewEncoder(Stdout)
-		enc.SetIndent("", "  ")
-		return enc.Encode(updates)
-	}
 
 	out, useColor := colorableOutput()
 
+	return printNetworkLockLog(updates, out, &nlLogArgs.json, useColor)
+}
+
+func printNetworkLockLog(updates []ipnstate.NetworkLockUpdate, out io.Writer, jsonSchema *jsonoutput.JSONSchemaVersion, useColor bool) error {
```

Comment:
I had to think for a bit whether `jsonSchema` can be a nil pointer; any reasons to not pass the value?
```suggestion
	return printNetworkLockLog(updates, out, nlLogArgs.json, useColor)
}

func printNetworkLockLog(updates []ipnstate.NetworkLockUpdate, out io.Writer, jsonSchema jsonoutput.JSONSchemaVersion, useColor bool) error {
```